### PR TITLE
Walking refactor

### PR DESCRIPTION
**Pull request by ftsell**
Since bitbots_quintic_walk grew over the last few competitions this pull request aims to improve code readability and enforce a more consistent coding style

### CODE REVIEW

**Comment by ftsell** [comment]
As I currently am not relly active let's do this PR now so we can continue working.

Also the things additional things which I wanted to do with this are probably not useful before the spline library

File: bitbots_quintic_walk/include/bitbots_quintic_walk/QuinticWalkingNode.hpp [hunk 1]
```diff
@@ -45,155 +45,175 @@ The original files can be found at:
...
-     */
-    void initializeEngine();
+        ros::Subscriber subCmdVel;
+        ros::Subscriber subRobState;
+        ros::Subscriber subJointStates;
+        ros::Subscriber subKick;
+        ros::Subscriber subImu;
+        ros::Subscriber subCopL;
+        ros::Subscriber subCopR;
+    };
```
Comment (SammyRamone) [inline bitbots_quintic_walk/include/bitbots_quintic_walk/QuinticWalkingNode.hpp]: why are you using a struct? What's the advantage?

Comment (ftsell) [reply bitbots_quintic_walk/include/bitbots_quintic_walk/QuinticWalkingNode.hpp]: That's a good question. There is no definitive reason. I just _felt_ it was somehow better have fewer direct instance variables but now I do agree that it is sort of useless

Comment (ftsell) [reply bitbots_quintic_walk/include/bitbots_quintic_walk/QuinticWalkingNode.hpp]: I removed the struct

File: bitbots_quintic_walk/CMakeLists.txt [hunk 2]
```diff
@@ -59,34 +68,12 @@ include_directories(
...
-)
-
-
 set(CODE_LIBRARIES
         urdfdom_world
         urdfdom_model
-)
-
-add_library(${PROJECT_NAME} ${SOURCES})
+        )
```
[the remark this answers is not in the document]
Comment (ftsell) [reply bitbots_quintic_walk/CMakeLists.txt]: fixed

**Comment by ftsell** [comment]
I removed commit 1386a76 which results in the pressure callback being implemented again.

I specifically decided against seperating needed and unneeded code in the callback because we are going to probably make changes to it again once we implement the Spline library and I don't want to spend the time twice.

**Pull request merged**